src/Bitmap/ImageHeader.h

```diff
@@ -42,8 +42,9 @@ struct ImageHeader
 	void VerifyValidBitCount() const;
 	static void VerifyValidBitCount(uint16_t bitCount);
 
-	std::size_t CalculatePitch() const;
-	static std::size_t CalculatePitch(uint16_t bitCount, int32_t width);
+	// Default pitch granularity is 4 bytes
```

Comment:
I had intentionally left this as interval. Perhaps it is a regional language different but interval seems a better and easier to understand choice than granularity to me. Being a comment, it probably isn't worth arguing over now that it has been changed.

Reply:
A disagreement in terminology might signal a disagreement about what is intended here.

I would define an interval as having a start and end point. Granularity is about the minimum spacing of values.

Here the pitch is rounded to some multiple of a certain value. That is defining the granularity of the value. As in, values are a multiple of 4 bytes. Is does not define the length of an interval as 4 bytes. For example, the spacing between scanlines could easily be 12 bytes, even though the granularity of that value is 4 bytes.

More deeply, I'm not sure I would really consider pitch to be an interval. It's more of a spacing between scanlines. If pitch were an interval, I'd be uncertain how to define the start and end points.

Looking back at `std::size_t CalculateDefaultPitch() const;`, I'm actually a little uncertain what the comment is trying to tell me, or why it is important. Is it trying to tell me the value is a multiple of 4 bytes? From the example test code, that would seem to be the case.

I vaguely recall some requirement that pitch is a minimum of 4 bytes, though can be a larger multiple of it.

I might also consider the phrasing:
Default pitch is the scanline byte width rounded up to a multiple of 4 bytes